content/hardware/01.mkr/03.carriers/mkr-iot-carrier-rev2/datasheet/datasheet.md

```diff
@@ -260,6 +260,8 @@ Detailed information on the pinouts are available in a separate document: <u>[MK
 
 ## Certifications
 
+Visit this page for the full [Product Compliance list](https://docs.arduino.cc/certifications/). 

```

Comment:
Hello @g-castronovo 

please, not hyperlinks hidden in in normal text in the case of the datasheet.

Been a document in a printed format, those links just disappear if you print it, at it is pretty common regarding this kind of technical and legal texts.

Therefore:

```suggestion
Visit the following website to learn more about the full Product Compliance of Arduino products: [Product Compliance list](https://docs.arduino.cc/certifications/)](https://docs.arduino.cc/certifications/). 
```

For the same reason, we also need to add a citation reference number and make sure all of them are in the correct order (from 1 to N, I know, it is little painful)